### ISSUE TITLE
Docker container reports errors on Apple silicon

### ISSUE THREAD

**Issue by christopherwharrop-noaa** [NOAA-GSL/ExascaleWorkflowSandbox#35]
The current docker-compose configuration contains a bug.  When test containers are launched on Mac laptops using the new Apple chips, the following error is emitted:

```
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```
The `docker-compose.yml` file likely needs the following added for each service listed (see [here](https://stackoverflow.com/questions/69054921/docker-on-mac-m1-gives-the-requested-images-platform-linux-amd64-does-not-m))

```
platform: linux/amd64
```

Since Docker only works on Linux systems for Github Actions, this is not currently testable in the automated CI.  However, it can be verified locally by a user running Docker Desktop on a Mac with a M1 or M2 processor.

**Comment by christopherwharrop-noaa** [comment]
@NaureenBharwaniNOAA - Please take a look at this and see if the proposed solution addresses the problem.

**Comment by NaureenBharwaniNOAA** [comment]
@christopherwharrop-noaa Looking at the article linked, I think this would be a useful starting point and trial to see if this proposed solution could fix our issue.

**Comment by christopherwharrop-noaa** [comment]
This issue requires use of an ARM machine for building the containers natively.  Until access to ARM is obtained for CI processing, this issue cannot be fixed.

**Comment by christopherwharrop-noaa** [comment]
This issue was addressed by #66
